### PR TITLE
Azure Theme Update (Color Pickers, Bug Fixes, Teaching Bubble)

### PR DESCRIPTION
**Pull request by chieftn**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes
Add color picker, rating, card styles.
Remove AzureMotion (not used)
Update color codes.
Slim down code on messageBar,
Bugfies on slider, detailslist, dialog.

#### Focus areas to test
NA

### CODE REVIEW

File: packages/azure-themes/src/azure/AzureThemeDark.ts [hunk 1]
```diff
@@ -78,10 +78,14 @@ export const AzureThemeDark: ITheme = createTheme({
   },
   palette: {
     themePrimary: DarkSemanticColors.controlOutlines.accent,
+    neutralPrimary: DarkSemanticColors.text.body,
     neutralDark: DarkSemanticColors.text.body,
-    neutralLight: DarkSemanticColors.item.hover, // shimmer elements
+    neutralLight: CommonSemanticColors.dividers.lineSeparator, //shimmer and document card
     neutralLighter: DarkSemanticColors.item.hover, // shimmer elements
```
Comment (Vitalius1) [inline packages/azure-themes/src/azure/AzureThemeDark.ts]: just curious what are the shimmer elements look like))) can you share a screenshot on Teams? 😄

File: packages/azure-themes/src/azure/styles/MessageBar.styles.ts [hunk 2]
```diff
@@ -2,201 +2,50 @@ import { IStyle } from 'office-ui-fabric-react';
 import { IMessageBarStyleProps, IMessageBarStyles, MessageBarType } from 'office-ui-fabric-react/lib/MessageBar';
 import { IExtendedSemanticColors } from '../IExtendedSemanticColors';
 
-const IconButtonStyles = (props: IMessageBarStyleProps): IStyle => {
```
Comment (Vitalius1) [inline packages/azure-themes/src/azure/styles/MessageBar.styles.ts]: Thanks a lot for this cleanup 👍

File: packages/azure-themes/src/azure/styles/Slider.styles.ts [hunk 3]
```diff
@@ -2,13 +2,14 @@ import { ISliderStyleProps, ISliderStyles } from 'office-ui-fabric-react/lib/Sli
 import * as StyleConstants from '../Constants';
 import { IExtendedSemanticColors } from '../IExtendedSemanticColors';
 
+const SLIDER_BOX_DIMENSION: number = 8;
```
Comment (Vitalius1) [inline packages/azure-themes/src/azure/styles/Slider.styles.ts]: 👍

**Comment by Vitalius1** [comment]
Restarting the build... seems odd that it fails on a test from OUFR that has nothing to do with your changes.
screen shot 2019-01-16 at 1 49 05 pm

**Comment by Vitalius1** [comment]
Build complains about the comments in `AzureThemeLight` and `AzureThemeDark`. They all have to start with a space: `// shimmer...` instead of `//shimmer...`
Here are the lines with the error:
screen shot 2019-01-16 at 4 14 06 pm

**Comment by dzearing** [comment]
Hmm. weird, maybe prettier didn't auto format?

File: packages/azure-themes/src/azure/styles/TeachingBubble.styles.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+          borderColor: semanticColors.inputBorder,
+          borderStyle: StyleConstants.borderSolid,
+          borderWidth: StyleConstants.borderWidth,
+          boxShadow: Depths.depth8
+        }
+      }
+    },
+    closeButton: {
+      color: semanticColors.bodyText,
+      margin: '1px',
```
Comment (KevinTCoughlin) [inline packages/azure-themes/src/azure/styles/TeachingBubble.styles.ts]:
```suggestion
      margin: 1,
```

Comment (chieftn) [reply packages/azure-themes/src/azure/styles/TeachingBubble.styles.ts]: We too are concerned about selector specificity.  We've had to get rather 'artful' in some case.  But there may be better ways to do the same thing.  Do you have a couple of standouts you encountered.  Anything dealing with buttons generally caught our attention.

**Comment by chieftn** [comment]
Looks like I am failing the travis CI build.  Do I need to invoke rush change from my branch?

**Comment by chieftn** [comment]
closing and reopening to hopefully correct build issue.

File: packages/azure-themes/src/azure/AzureThemeDark.ts (hunk 1 above)
Comment (dzearing) [reply packages/azure-themes/src/azure/AzureThemeDark.ts]: You should be able to use the pr links I think! :)

http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/7681/merge/fabric-website-resources/dist/index.html#/examples/shimmer

**Pull request merged**